### PR TITLE
Update settings in teams-calling-policy.md

### PR DESCRIPTION
**Pull request by dariomws**
Closes https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/7904.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks

File: Teams/teams-calling-policy.md [hunk 1]
```diff
@@ -86,21 +90,29 @@ This setting enables inbound calls to be routed to delegates, allowing delegates
...
-### Busy on busy is available when in a call
+### Music on hold for PSTN callers
+
+This setting allows you to enable or disable music on hold when a PSTN caller is placed on hold. It's enabled by default. This setting doesn't apply to call park and boss delegate features. Valid options are:
+
+- **Enabled:** Music on hold is always available when a PSTN caller is placed on hold.
+- **Disabled:** Music on hold is not available when a PSTN caller is placed on hold.
+- **User controlled:** Users can determine whether they want music on hold to be available.
+
+### Busy on busy when in a call
```
Comment (JohanFreelancer9) [inline Teams/teams-calling-policy.md:101]:
```suggestion
### Busy on Busy when in a call
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor suggestion.

File: Teams/teams-calling-policy.md (hunk 1 above)
Comment (dariomws) [reply Teams/teams-calling-policy.md:101]: Let's leave it as it is on the Admin Center.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@tonysmit PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request closed without merging**